src/components/conversation/conversation-send-message-box/ConversationSendMessageBox.vue

```diff
@@ -106,6 +106,7 @@ export default {
 
 <style lang="scss" scoped>
 @import '../../../assets/style/default';
+@import '../../../assets/style/index';
```

Comment:
Why didn't use the default scss?
You have also variable there
Mixing both could provoke conflict in classes and overwriting